pkgs/tools/text/ocrmypdf/default.nix

```
@@ -93,6 +96,8 @@ in buildPythonApplication rec {
     and not test_old_unpaper'
   '';
 
+  makeWrapperArgs = [ "--prefix" "PATH" ":" "${stdenv.lib.makeBinPath [ ghostscript jbig2enc pngquant python3Packages.setuptools qpdf tesseract4 unpaper ]}" ];
```

```suggestion
  makeWrapperArgs = [ "--prefix PATH ":" ${stdenv.lib.makeBinPath [ ghostscript jbig2enc pngquant qpdf tesseract4 unpaper ]} ];
```

Quoting and we don't need to add setuptools here.
Apparently setuptools is no longer propagated so you need to add it to `propagatedBuildInputs`.
(and drop the propagatedNativeBuildInputs)

---

setuptools should only be need when building the wheel or installing, I don't think it's needed at run time at all. In fact, it seems to work when I remove it

---

It was failing in nix-review without me adding setuptools. I'm just as confused as you.

```
Traceback (most recent call last):
  File "/nix/store/r29hhnh1djfa4kbny14c07qryyhmn9bg-ocrmypdf-8.2.3/bin/.ocrmypdf-wrapped", line 7, in <module>
    from ocrmypdf.__main__ import run_pipeline
  File "/nix/store/r29hhnh1djfa4kbny14c07qryyhmn9bg-ocrmypdf-8.2.3/lib/python3.7/site-packages/ocrmypdf/__init__.py", line 18, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```